types/office-js-preview/index.d.ts

```diff
@@ -11946,6 +11946,13 @@ declare namespace Office {
          *
          * **Important**: In Outlook on the web, `setSignatureAsync` only works on messages.
          *
+         * **Important**: If your add-in implements the 
+         * {@link https://docs.microsoft.com/office/dev/add-ins/outlook/autolaunch | event-based activation feature using `LaunchEvent` in the manifest},
+         * and calls `setSignatureAsync` in the event handler, the following behavior applies.
+         *
+         * - When the user composes a new item (including reply or forward), the signature is set but doesn't modify the form. This means
+         * if the user closes the form without making other edits, they won't be prompted to save changes and a draft won't be created.
```

Comment:
Sorry, I just realized that my comment about not creating a draft only applies to win32, since OWA has the auto-save feature, and a draft is automatically created. So I think we should either call out that this is for win32 (or not mention drafts at all).

Reply:
@emtencza Thanks for the clarification. I'll use the original text.